### ISSUE TITLE
Script Alert when using size marks

### ISSUE THREAD

**Issue by jpepp845** [romashamin/Size-Marks-PS#31]
When trying to run the size marks script I get the attached error.  I also included the photoshop version information. This is the latest version of photoshop released in late June. 
![untitled-1](https://cloud.githubusercontent.com/assets/20286719/16567338/918a8e18-41eb-11e6-9dc2-2885c6939a00.jpg)

**Comment by jpepp845** [comment]
I seem to be getting the same error Pay4m is receiving.

**Comment by jpepp845** [comment]
![sm_2](https://cloud.githubusercontent.com/assets/20286719/17634164/3f55072c-609f-11e6-8d36-d29062330f64.jpg)

Still having the same issue in latest release of Photoshop. It runs the script and about half way through, then error shows up.  When I click ok the text layer with the measurement shows up but the guide lines do not.

**Comment by romashamin** [comment]
@jpepp845 thanks for the issue.

First, please be sure you didn’t [rename downloaded ZIP to jsxbin](https://github.com/romashamin/Size-Marks-PS/issues/32#issuecomment-234915592). It’s the most frequent cause of these kinds of errors.

**Comment by jpepp845** [comment]
Correct. I saved the zip, unzipped the files and placed the Size Marks.jsxbin file in the directory you outlined.
